test/functional/fixtures/api/es-next/type/test.js

```diff
@@ -0,0 +1,37 @@
+var expect = require('chai').expect;
+
+// NOTE: we run tests in chrome only, because we mainly test server API functionality.
+// Actions functionality is tested in lower-level raw API.
+describe('[API] t.typeText()', function () {
+    it('Should type text in input', function () {
+        return runTests('./testcafe-fixtures/type-test.js', 'Type text in input', { shouldFail: true, only: 'chrome' })
```

Comment:
You can now use `Hybrid` + chai assertions. No need to throw in tests which should complete sucesfully.